etc/profile-a-l/loupe.profile

```diff
@@ -0,0 +1,48 @@
+# Firejail profile for loupe
+# Description: Profile for modern GNOME Image Viewer program
+# This file is overwritten after every install/update
+# Persistent local customizations
+include loupe.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.local/share/Trash
+noblacklist ${HOME}/.Steam
+noblacklist ${HOME}/.steam
+
+noblacklist /usr/libexec/glycin-loaders
+noblacklist /usr/share/glycin-loaders
+
+#include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-write-mnt.inc
+
+include whitelist-runuser-common.inc
+#include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+machine-id
+net none
+nodvd
+nogroups
+noinput
+nonewprivs
+noroot
+nosound
+notv
+nou2f
+novideo
+protocol unix,netlink
```

Comment:
No `seccomp`?

Reply:
glycin already applies a seccomp filter https://gitlab.gnome.org/sophie-h/glycin#sandboxing-and-inner-workings

Reply:
Cool. Perhaps a note to that effect might help avoiding someone adding seccomp in the future. But I leave that up to you. Nice seeing you here BTW.